bin/plugin/commands/changelog.js

```diff
@@ -33,6 +33,47 @@ const manifest = require( '../../../package.json' );
  * @property {string}  milestone Milestone title.
  */
 
+/**
+ * Changelog normalization function, returning a string to use as title, or
+ * undefined if entry should be omitted.
+ *
+ * @typedef {(text:string,issue:IssuesListForRepoResponseItem)=>string|undefined} WPChangelogNormalization
+ */
+
+/**
+ * Mapping of label names to grouping heading text to be used in release notes,
+ * intended to be more readable in the context of release notes. Also used in
+ * merging multiple related groupings to a single heading.
+ *
+ * @type {Record<string,string>}
+ */
+const LABEL_GROUP_TITLES = {
+	Bug: 'Bug Fixes',
+	Regression: 'Bug Fixes',
+	Feature: 'Features',
+	Enhancement: 'Enhancements',
+	'New API': 'New APIs',
+};
+
+/**
+ * Order in which to print group titles. A value of `undefined` is used as slot
+ * in which unrecognized headings are to be inserted.
+ *
+ * @type {Array<string|undefined>}
+ */
+const GROUP_TITLE_ORDER = [
+	'Features',
+	'Enhancements',
+	'New APIs',
+	'Bug Fixes',
+	'Experiments',
+	'Code Quality',
+	'Documentation',
+	'Project Management',
+	undefined,
```

Comment:
What is undefined here?

Reply:
> What is undefined here?

It's a little opaque, but explained in the comment:

https://github.com/WordPress/gutenberg/blob/a24fe1117b5bc680ea1d686fb46a5efb7fd467de/bin/plugin/commands/changelog.js#L59-L60

Reply:
Do you think we can use the "grouping" like here https://github.com/WordPress/gutenberg/pull/22601/files#diff-ea435cc89a172cc49a2ab3b0ff264cc1R52 to automatically move these "undefined" items under "Various" instead?

Reply:
The thinking is to try to avoid having to have built-in awareness of every possible heading, while at the same time wanting "Various" as both known and intending to be the very-last heading.

Reply:
Oh, it's more clear now, thanks :)